internal/ignition/ignition.go

```diff
@@ -839,18 +843,26 @@ func uploadToS3(ctx context.Context, workDir string, cluster *common.Cluster, s3
 	return nil
 }
 
-func parseIgnitionFile(path string) (*config_31_types.Config, error) {
-	configBytes, err := ioutil.ReadFile(path)
+func ParseTo32(content []byte) (*config_32_types.Config, error) {
+	configv32, _, err := config_32.Parse(content)
 	if err != nil {
-		return nil, errors.Errorf("error reading file %s: %v", path, err)
+		configv31, _, err := config_31.Parse(content)
+		if err != nil {
+			return nil, errors.Errorf("error parsing ignition: %v", err)
```

Comment:
Can it contain sensitive information?

Reply:
I don't think so. But this is no different to what we were doing before ... so :man_shrugging: